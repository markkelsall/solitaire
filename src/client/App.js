import React, { Component } from "react";

class App extends Component {

  render() {
    return (
      <div>
          <h1 className="hello">Hello! 1</h1>
      </div>
    );
  }
}

export default App;
