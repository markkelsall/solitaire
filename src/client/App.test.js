import React from "react";
import renderer from 'react-test-renderer';

import App from "./App";

describe("App/>", () => {
    it("should render", () => {
        const component = renderer.create(<App />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});