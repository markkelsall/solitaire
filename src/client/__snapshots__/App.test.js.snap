// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App/> should render 1`] = `
<div>
  <h1
    className="hello"
  >
    Hello! 1
  </h1>
</div>
`;
